### PR TITLE
feat: kickstart responsive ux

### PR DESCRIPTION
**Pull request by vbstreetz**
- https://github.com/Synthetixio/issues/issues/44
- phase 1 only includes basic responsive updates that make the dapp usable on xs devices. more updates to be done in subsequent commits.

### CODE REVIEW

File: sections/shared/Layout/SideNav/MobileSideNav.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+	return (
+		<Container data-testid="sidenav" isShowing={isShowingMobileSideNav}>
+			<StakingLogoWrap>
+				{isShowingSubMenu ? (
+					<Svg src={BackIcon} onClick={clearSubMenuConfiguration} />
+				) : (
+					<Link href={ROUTES.Home}>
+						<>
+							<Svg src={StakingLogo} />
```
Comment (JChiaramonte7) [inline sections/shared/Layout/SideNav/MobileSideNav.tsx:33]: Thinking maybe we could add some padding to this. Kinda hard to tap the close button sometimes on mobile.

File: sections/layer2/components/common.ts [hunk 2]
```diff
@@ -34,7 +34,7 @@ export const InputContainer = styled(FlexDivColCentered)`
 	position: relative;
 	width: 100%;
 	padding: 16px;
-	height: 600px;
+	/* height: 600px; */
```
Comment (JChiaramonte7) [inline sections/layer2/components/common.ts:37]: did you intend to keep this?

File: sections/staking/components/common.ts [hunk 3]
```diff
@@ -26,7 +26,7 @@ export const InputContainer = styled(FlexDivColCentered)`
 	width: 100%;
 	padding: 16px;
 	margin-bottom: 24px;
-	height: 600px;
+	/* height: 600px; */
```
Comment (JChiaramonte7) [inline sections/staking/components/common.ts:29]: same here

File: containers/SideNav/SideNav.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+export default createContainer(Container);
+
+function Container() {
+	const [isShowingMobileSideNav, setIsShowingMobileSideNav] = useState(false);
+	const [subMenuConfiguration, setSubMenuConfiguration] = useState<SubMenuConfiguration>({
+		routes: null,
+		topPosition: 0,
+	});
+	const isShowingSubMenu = !!subMenuConfiguration?.routes ?? false;
```
Comment (JChiaramonte7) [inline containers/SideNav/SideNav.tsx:19]: if subMenuConfiguration.routes is null/undefined won't `!!` already make it false?

File: sections/dashboard/PossibleActions/Layouts/LayoutLayerOne.tsx [hunk 5]
```diff
@@ -287,15 +285,41 @@ const LayoutLayerOne: FC = () => {
 				link: ROUTES.Earn.sCOIN_LP,
 				isDisabled: lpData[LP.BALANCER_sCOIN].APR === 0,
 			},
-		];
+		].map((cell, i) => ({ ...cell, gridArea: `tile-${i + 1}` }));
```
Comment (JChiaramonte7) [inline sections/dashboard/PossibleActions/Layouts/LayoutLayerOne.tsx:288]: 🎉

File: sections/delegate/constants.tsx [hunk 6]
```diff
@@ -1,3 +1 @@
-export const FORM_COL_WIDTH = 382;
-
-export const TABLE_COL_WIDTH = 495;
+export const NOOP = 1;
```
Comment (JChiaramonte7) [inline sections/delegate/constants.tsx:1]: is this used? 

also i thought a no-op would be more of a function like `() => {}`

File: sections/loans/components/ActionBox/ActiveBorrowsTab/LoanList.tsx [hunk 7]
```diff
@@ -22,13 +22,13 @@ const LoanList: React.FC<LoanListProps> = ({ actions }) => {
...
 
-	const columns = React.useMemo(
+	const desktopColumns = React.useMemo(
 		() => [
 			{
 				Header: <>{t('loans.tabs.list.types.debt')}</>,
 				accessor: 'debt',
 				sortable: true,
-				width: LARGE_COL_WIDTH,
+				// width: LARGE_COL_WIDTH,
```
Comment (JChiaramonte7) [inline sections/loans/components/ActionBox/ActiveBorrowsTab/LoanList.tsx:31]: maybe remove these

File: sections/shared/Layout/SideNav/MobileSideNav.tsx (hunk 1 above)
Comment (vbstreetz) [reply sections/shared/Layout/SideNav/MobileSideNav.tsx:33]: Gotcha. Done.

File: sections/layer2/components/common.ts (hunk 2 above)
Comment (vbstreetz) [reply sections/layer2/components/common.ts]: Done.

File: containers/SideNav/SideNav.tsx (hunk 4 above)
Comment (vbstreetz) [reply containers/SideNav/SideNav.tsx]: Gotcha 👍🏽

File: sections/dashboard/PossibleActions/Layouts/LayoutLayerOne.tsx (hunk 5 above)
Comment (vbstreetz) [reply sections/dashboard/PossibleActions/Layouts/LayoutLayerOne.tsx:288]: 👍🏽

File: sections/delegate/constants.tsx (hunk 6 above)
Comment (vbstreetz) [reply sections/delegate/constants.tsx]: Gotcha. Main intent was to keep the file but I've removed it from now.

File: sections/loans/components/ActionBox/ActiveBorrowsTab/LoanList.tsx (hunk 7 above)
Comment (vbstreetz) [reply sections/loans/components/ActionBox/ActiveBorrowsTab/LoanList.tsx]: Done.

File: sections/staking/components/common.ts (hunk 3 above)
Comment (vbstreetz) [reply sections/staking/components/common.ts]: Done.

**Review by JChiaramonte7 — APPROVED** [review]

**Review by clementbalestrat — APPROVED** [review]
Awesome work @vbstreetz !

**Pull request merged**